### ISSUE TITLE
Produce translated flows and upload to RapidPro server to check that files are valid

### ISSUE THREAD

**Issue by fagiothree** [IDEMSInternational/parenttext-china#18]
The first round of translation has been completed. We can now produce a first version of translated flows and upload them to the test workspace on the Chinese RapidPro server https://www.plhparenting.cn/org/home/ to check that all files are valid: the json might be invalid if variable names contain chinese characters. Since the server has time out issues when importing flows, the suggested strategy to import is to produce subset of flows using tags and to import in the following order:
1. safeguarding
2.  modules
3. all other content tags
4. menu + delivery + onboarding
it's possible that flows will need to be imported twice (1-2-3-4-1-2-3)

**Comment by edmoss345** [comment]
Produced and uploaded the flows in the order suggested and didnt run into any problems. However, some files are missing some translations, specifically:

- safeguarding - 74 words
- home_activity_checkin - 424 words

I tried to investigate the safeguarding but then started getting errors (I think @fagiothree you may have been editing this sheet at the time). For the home_activity_checkin, I can see that there are just some strings that arent in the files on crowdin, as summarised in this PR

https://github.com/IDEMSInternational/plh-digital-content/pull/560

Not sure if I somehow missed these when I did the initial upload, or if changes have been made in the meantime. But I am assuming it is ok to merge this PR an then we just need to let the China team know that there is more content to translate?

**Comment by fagiothree** [comment]
For **safeguarding**, there was a small issue in Crowdin that has now been resolved and the missing strings were included in the latest pull request.

For **home_activity_checkin** the missing translations were a result of the home_activity_checkin template still including whatsapp templates, not needed for China. I created a new version of the template in https://docs.google.com/spreadsheets/d/1CoYl7lpd6w2xdZzE-Crt-pZT9M5dTHYFm9sWEcX5QAs/edit#gid=164262199, removing references to WA. There is still one missing string (You’re doing great towards your goal, @results.name.) that came from introducing webhook calls in the delivery flows for getting goal/module data, so I updated the file in Crowdin with https://github.com/IDEMSInternational/plh-digital-content/pull/562
